examples/configs/dpo.yaml

```diff
@@ -28,6 +28,7 @@ checkpointing:
   save_period: 50
 
 policy:
+  training_backend: "hf"
```

Comment:
we've deprecated this arg; it's all inferred from the presence of 'megatron_cfg['enabled']=true'/dtensor etc.